functions/RemotePlayClientScripts/remotePlaySteamLink.sh

```diff
@@ -2,7 +2,7 @@
 
 # Variables
 SteamLink_emuName="SteamLink"
-SteamLink_emuType="FlatPak"
+SteamLink_emuType="$emuDeckEmuTypeFlatpak"
```

Comment:
If we are using flatpaks, probably should use the install methods we have for them already too, instead of rewriting it in the script. I'd like to see it implemented in this PR but it can wait until later as well.

i think this would be a good step towards more reuse.